brink.sh

```diff
@@ -86,7 +86,7 @@ ARCH='not-detected-yet'
 PYTHON_CONFIGURATION='NOT-YET-DEFINED'
 PYTHON_VERSION='not.defined.yet'
 PYTHON_PLATFORM='unknown-os-and-arch'
-PYTHON_NAME='python2.7'
```

Comment:
this is ok, but we will need to add this to brink.conf in order to allow a "smooth" transition.

Reply:
> this is ok, but we will need to add this to brink.conf in order to allow a "smooth" transition.

Do you mean using https links here?

I'm confused because the line below (not shown here) seem to be more relevant than the line above…

Reply:
What I wanted to say is that we make sure we don't hardcode the python3 into brink.sh

We should allow running brink.sh on both python 2 and python3 ... but with different brink.conf

But is ok to start with hardcoded python3 and later we can see how to change it if we found issues

Reply:
On Windows, the binary in the upstream packages is named `python.exe` and there's no `python3.exe`.

On Linux/macOS, after the build, there's `python3` and `python3.9`, but no `python` binary.

For now, I'm leaving Windows files alone and symlinking on non-Windows platforms to get `python`.

Reply:
On Linux it is named python3 as on many distro there is both py2 and py3 installed on the same system.

And python is used for python2.

But for pythia we are on our own and there should be a problem to name it just python, even if its py3

But at least for chevah/server we will need to rename it to bin/python as I know at least one customer which is not using our default init script, but instead is using a custom script.

also, the init scripts / unit files  for chevah/server are not updated with each update... so at least for chevah/server bin/python is our fixed entry point.

But as I mentioned above, I can update the chevah/server pavemement.py script to do the rename.

Reply:
OK, but what is the problem then? On Windows I left things as set upstream, and `python` is used. While on Linux there a symlink to have `python` available too, besides the default `python3` binary.

Reply:
Hhmmm, I'm not sure where I've seen the `python` binary missing… It's present on all OS'es, from what I can tell.